_includes/footer.html

```diff
@@ -1,49 +1,47 @@
 <footer class="footer hidden-print">
-  <div class="section section-footer-content">
-    <div class="container">
-      {% if page.footer_heading %}
-      <div class="row row-call-to-action">
-        <div class="col-xs-12 text-center">
-          <h2 class="h1 no-anchor">{{ page.footer_heading }}</h2>
-          <p><a href="/checkout/" class="btn btn-info js-checkout-link" role="button" ga-on="click" ga-event-category="footer-{{ page.path | slugify }}" ga-event-action="get-started">Get Started</a></p>
+    <div class="section section-footer-content">
```

Comment:
It's a bit hard to see from this diff view... What actually changed in the footer? Is this just a whitespace change? As mentioned earlier, we probably don't need any footer changes yet.

Reply:
It's just a white space change. I already reverted the changes to what we have on the website

Reply:
I'm still seeing a diff in GitHub for the footer. Did this not get reverted?

Reply:
This has been reverted. It's just some whitespaces and minor code formatting changes.